v63003/outref/gtm8680.txt

```diff
@@ -0,0 +1,19 @@
+# Running test with 1 process
+# Locking ^a in child function and attempting to lock ^a(i) in parent
+##TEST_AWK# Max Iterations in a second=[0-9]*
+##TEST_AWK# Min Iterations in a second=[0-9]*
+# No Significant Slowdown Experienced
+%YDB-I-LOCKSPACEINFO, Region: DEFAULT: processes on queue: 0/240000; LOCK slots in use: 100001/181132; name space not full
+
+# Running test with 100 processes, each holding 1000 locks
+# Locking ^a(1)-^a(100000) in child processes, attempting to lock ^a(i) in parent
+##TEST_AWK# Max Iterations in a second=[0-9]*
+##TEST_AWK# Min Iterations in a second=[0-9]*
+# No Significant Slowdown Experienced
+##TEST_AWK%YDB-I-LOCKSPACEINFO, Region: DEFAULT: processes on queue: 0/240000; LOCK slots in use: [0-9]*/181132; name space not full
+
+Database file ##TEST_PATH##/mumps.dat now has lock space 65536 pages
```

Comment:
Add a comment here that we want to test maximum lock space supported is now bumped from 65536 to 262144